src/extras/gizmo/axis-shapes.js

```diff
@@ -51,7 +51,7 @@ const SHADER = {
             // store z/w for later use in fragment shader
             vZW = gl_Position.zw;
             // disable depth clipping
-            gl_Position.z = 0.0;
+            // gl_Position.z = 0.0;
```

Comment:
Delete?

Reply:
It looks cleaner when clipping with this so would prefer to have it in but the selection area and rendered area do not align